src/components/MenuBar/ExportLogsButton.tsx

```diff
@@ -49,9 +49,7 @@ const ExportLogsButton = () => {
                         "primary"}
                 >
                     {EXPORT_LOG_PROGRESS_VALUE_MAX === exportProgress ?
-                        <DownloadIcon
```

Comment:
We probably don't want to remove this, right?

With `sx={{fontSize: "14px"}}`:
![image](https://github.com/user-attachments/assets/cdc19c4b-dd6a-4298-8a57-7560de2aeda7)

Without:
![image](https://github.com/user-attachments/assets/ca208d2d-78b1-48ad-a31e-57fbad501181)

Reply:
My appologies. I wasn't paying attention to the icon when download finishes.